README.md

```diff
@@ -60,3 +74,38 @@ Available package names are:
 - e2e-tests
 - integration-tests
 - quiet (frontend)
+
+----
+
+## Locally linking packages (mobile)
```

Comment:
Is this important to get things running on mobile? Should it be part of the steps?

Or is this just if you're hacking on packages locally?

Reply:
It's a useful tip but it's not a step of environment setup